ads/yieldmo.js

```diff
@@ -28,7 +28,8 @@ export function yieldmo(global, data) {
   ymElem.dataset['ampEnabled'] = true;
   global.document.getElementById('c').appendChild(ymElem);
 
-  const ymJs = 'https://static.yieldmo.com/ym.amp1.js';
+  var swimLane = Math.round( 5 * Math.random() / 3 );
```

Comment:
is it intended to do 5/3?
This will give you uneven distribution among 0.js, 1.js, 2.js